scipy/integrate/_ode.py

```diff
@@ -450,7 +450,7 @@ def set_solout(self, solout):
                 self._integrator.reset(len(self._y), self.jac is not None)
         else:
             raise ValueError("selected integrator does not support solout,"
-                            + " choose another one")
+                             + " choose another one")
```

Comment:
@perimosocordiae I didn't add the `+` operator, but I can remove it if you'd like/